services/galley/galley.integration.yaml

```diff
@@ -27,8 +27,8 @@ settings:
   intraListing: false
   conversationCodeURI: https://app.wire.com/join/
 
-logLevel: Info
-logNetStrings: false
+logLevel: Debug
+logFormat: JSON
```

Comment:
i see that you wanted to test this, but it's a bit odd to do have gally log in json, and everybody else log the old way.  Could you revert this change?